pyccel/codegen/printing/fcode.py

```diff
@@ -1494,7 +1494,6 @@ def _print_BindCFunctionDef(self, expr):
         interfaces = '\n'.join(self._print(i) for i in expr.interfaces)
         arg_code  = ', '.join(self._print(i) for i in chain( arguments, results ))
         imports   = ''.join(self._print(i) for i in expr.imports)
-        imports += 'use, intrinsic :: ISO_C_BINDING'
```

Comment:
I thought that we needed this in order to get the integer kind type parameters. What has changed?